### DISCUSSION TITLE
When to update the LCD firmware

### DISCUSSION THREAD

**Discussion by pedrolamas** [Jyers/Marlin#537] [Q&A] [answered]
First of all, thank you for the awesome work on this firmware @Jyers!

Is there a way that we can know for sure if a specific release needs to also have the LCD firmware updated? I don't see any information in terms of this on the release notes and thus why I'm now asking!

**Comment by Tuncay-Ayhan** [comment, marked as answer]
It is not needed. But only when you want the custom icons.

https://github.com/Jyers/Marlin/wiki/Configuring-&-Flashing

```
## Screen Flashing
From v1.2.0 on, there is now the option to flash the screen to enable the use of custom icons specifically for this firmware. The organization of the flashing tools and assets is still underway and this section will be updated to detail the process for flashing your display with a new DWIN_SET.

### Note: Screen flashing will never be required, custom icons will simply be substituted with existing ones if you have not flashed your screen
```

**Reply by pedrolamas** [reply]
Seems I missed that, thank you for pointing it out @Tuncay-Ayhan! 🙂
